### ISSUE TITLE
Use ember-auto-import 

### ISSUE THREAD

**Issue by willviles** [willviles/ember-useragent#29]
Moving away from [ember-cli-cjs-transform](https://github.com/rwjblue/ember-cli-cjs-transform) for importing `ua-parser-js` module in favour of [ember-auto-import](https://github.com/ef4/ember-auto-import).

Unsure of the edge cases in using ember-auto-import in addons at the moment, so help wanted.

Try out the branch:

```
npm i -D git://github.com/willviles/ember-useragent.git#feat/ember-auto-import
```

or

```
yarn add --dev git://github.com/willviles/ember-useragent.git#feat/ember-auto-import
```

**Comment by mminkoff** [comment]
@willviles I believe those URLs/commands need to be `#feat/ember-auto-import`.

**Comment by willviles** [comment]
Thanks @mminkoff!

**Comment by buschtoens** [comment]
@willviles So far the branch is working fine for us. I think it's time to release it 😄

**Comment by nickschot** [comment]
Yes please! :D

**Comment by willviles** [comment]
Thanks for the input everybody - released in `0.9.0`!  🎉
